networks/motion/CNN2D/motion_all_CNN2D.py

```diff
@@ -128,6 +128,15 @@ def createModel(patchSize, architecture='new'):
 
 def fTrain(X_train, y_train, X_test, y_test, sOutPath, patchSize, batchSizes=None, learningRates=None, iEpochs=None):
     # grid search on batch_sizes and learning rates
+    # parse inputs
+    batchSizes = 64 if batchSizes is None else batchSizes
```

Comment:
that are still the old version. It should be:
`batchSize = [64] if batchSize is None else batchSize
learningRate = [0.01] if learningRate is None else learningRate
iEpochs = 300 if iEpochs is None else iEpochs`